SU-SWS/jumpstart_ui
# D8CORE-1010: Update all the things.

# READY FOR REVIEW

# Summary
- Updates all the NPM packages and required configuration
- Updates Decanter to V6
- Updates components with the new work in the updated branch

# Needed By (Date)
- End of the week would be nice

# Urgency
- Med-low

# Steps to Test

1. Review code
2. Check out this branch on your local build
3. Clear caches
4. Review /patterns for new variants
5. Run tests

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- D8CORE-1010
- FYI: @pookmish @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
